packages/wells/src/client/api/wells.ts

```diff
@@ -236,4 +250,71 @@ export class Wells extends AssetsAPI {
 
     return await this.getByIds(assetIds);
   };
+
+  public filterPolygonSearch = async ({
+    geometry,
+    crs = 'epsg:4326',
+    outputCrs = 'EPSG:4326',
+    limit = 1000,
+    name,
+    id,
+    data_source,
+    operator,
+    field,
+    block,
+    customFilter = undefined,
+  }: {
+    geometry: string | GeoJson;
+    crs?: string;
+    outputCrs?: string;
+    limit?: number;
+    name?: string;
+    id?: number;
```

Comment:
I know we've done that in the Cognuit api, so can't blame, but is it kind of weird to have id in a list filter? It may confuse the user. If they have the id, they could just call a "retrieve" method